lang/rs/bindings/cne-sys/build.rs

```diff
@@ -53,6 +53,11 @@ fn main() {
     println!("cargo:rustc-link-search=native={}", libcndp_so_path);
     println!("cargo:rustc-link-lib=cndp");
 
+    // CNDP pmd_af_xdp static library should be linked as whole archive.
+    // This will avoid preload PMD (eg: LD_PRELOAD=libpmd_af_xdp.so) while running
+    // CNDP CNE Rust binaries.
+    println!("cargo:rustc-link-lib=static:-bundle,+whole-archive=pmd_af_xdp");
```

Comment:
When using --whole-archive for pmd_af_xdp and you stated this is a static library, why do we need to build shared libraries or was the static text a typo?

Reply:
whole-archive is linked as static only for pmd_af_xdp. Rest of the CNDP CNE libraries are linked as shared library. So only pmd_af_xdp will be part of Rust binary. That way the binary size can be kept small.